### DISCUSSION TITLE
how to use backchannel to revoke token?

### DISCUSSION THREAD

**Discussion by bulolo** [ory/hydra#3487] [Q&A]
### Preflight checklist

- [X] I could not find a solution in the existing issues, docs, nor discussions.
- [X] I agree to follow this project's [Code of Conduct](https://github.com/ory/hydra/blob/master/CODE_OF_CONDUCT.md).
- [X] I have read and am following this repository's [Contribution Guidelines](https://github.com/ory/hydra/blob/master/CONTRIBUTING.md).
- [X] This issue affects my [Ory Network](https://www.ory.sh/) project.
- [X] I have joined the [Ory Community Slack](https://slack.ory.sh).
- [X] I am signed up to the [Ory Security Patch Newsletter](https://ory.us10.list-manage.com/subscribe?u=ffb1a878e4ec6c0ed312a3480&id=f605a41b53).

### Describe the bug

i can find that from 2.1.0,we can revokeOAuth2LoginSessions by sid,but revokeOAuth2LoginSessions did not revoke token

any plan on revokeOAuth2ConsentSessions by sid,so that we can revoke tokens,

or any idea that can revoke token on backchannel logout?

### Reproducing the bug

backchannel revoke tokens

### Version

2.1.0

### On which operating system are you observing this issue?

Linux

### In which environment are you deploying?

Kubernetes with Helm

**Comment by prigaux** [comment]
When you "acceptOAuth2LoginRequest", you must set "remember". 
Otherwise "revokeOAuth2LoginSessions" by "sid" does not do anything.

Cf "HandleHeadlessLogout" which calls "GetRememberedLoginSession" in https://github.com/ory/hydra/blob/a7579b8d1dff413f1dd588e53020ba734cc47c2e/consent/strategy_default.go#L1102

On this subject, see also https://github.com/ory/hydra/issues/3132
